bundles/org.codechecker.eclipse.plugin/src/org/codechecker/eclipse/plugin/config/CommonGui.java

```diff
@@ -240,7 +244,8 @@ public void run() {
             }
         });
         if (!globalGui) {
-            useGlobalSettings = config.isGlobal();
+            //TODO
```

Comment:
Please add a bit more verbose comment what to do later here.

Reply:
Todo to remove after refactor. Thanks for the notice.